### ISSUE TITLE
replace beta channel with next channel in npm

### ISSUE THREAD

**Issue by KazuCocoa** [appium/appium#16454]
https://www.npmjs.com/package/appium has `next` and `beta`/`rc` channel.
Appium 1 is going to be EOL and 2 is coming, so probably we can start pushing current `next` channel (appium2) to `beta` channel as well?

Appum 2.0 is probably not `rc` , so the `rc` channel should keep current 1.22.2-rc.0 for a while though.

**Comment by jlipps** [comment]
Yes, I agree, let's make `appium@beta` point to latest 2.0 release as well. We can probably update references to `next` in the docs with a quick find/replace too.

**Comment by mykola-mokhnach** [comment]
Closed as obsolete
